README.md

```diff
@@ -174,11 +174,13 @@ You can execute DSpot using the maven plugin. You can use this plugin on the com
 mvn eu.stamp-project:dspot-maven:LATEST:amplify-unit-tests
```

Comment:
What can be done here, is removing the version of the command line. In this way, maven will look up in the `pom.xml` if a version is specified, otherwise, it will look up in repository, _.e.g_ maven central for latest.

To conclude, I suggest to replace:
`mvn eu.stamp-project:dspot-maven:LATEST:amplify-unit-tests`
by
`mvn eu.stamp-project:dspot-maven:amplify-unit-tests`

And the line that specifies that you need to replace LATEST by the current latest version.

WDYT?

Reply:
That's fine with me. Indeed, it simplifies it as user doesn't have to care about what is the latest version.